### PR TITLE
Healthy thinking report icon list

### LINKED ISSUE #329: Refactor image-collage block to support 3 types of icon lists
1. `Icon-caption` variation
Mobile:
image

Tablet:
image

Desktop:
image

2. `Icon-title-text` variation

Mobile:
image

Tablet:
image

Desktop:
image

3. `Icon-title-list` variation

Mobile:
image

Tablet:
image

Desktop:
image

### PR DESCRIPTION
**Pull request by JiangLong2019**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #329 

Test URLs:
- Original: https://www.sunstar.com/healthy-thinking-report/oral-survey-2021
- Before: https://main--sunstar--hlxsites.hlx.live/healthy-thinking-report/oral-survey-2021
- After: https://healthy-thinking-report-icon-list--sunstar--hlxsites.hlx.live/healthy-thinking-report/oral-survey-2021

- [ ] New Blocks introduced in this PR
      If yes, please provide details below
**Block Name** : For e.g. cards
- [ ] Documented in sidekick library

- [x] New variations introduced in this PR
**Variation Name** :  Image-Collage(icon-caption), Image-Collage(icon-title-text), Image-Collage(icon-title-list) 
- [x] Documented in sidekick library

- [ ] New mixins introduced in this PR
**Mixin Name** :  For e.g. compact, white
- [ ] Documented in sidekick library

### CODE REVIEW

**Review by dnbute — COMMENTED** [review]
Do you think we could replace the authoring experience for the first collage with something like this? We would also leverage the caption autoblocking this way and it might be easier for the authors to visualise what they're going for

https://adobe.sharepoint.com/sites/HelixProjects/_layouts/15/doc.aspx?sourcedoc={e5317fd9-d72b-4045-9156-b442183640e3}&action=edit

**Comment by JiangLong2019** [comment]
>Do you think we could replace the authoring experience for the first collage with something like this? We would also leverage the caption autoblocking this way and it might be easier for the authors to visualise what they're going for

@dnbute It's a good idea to let the author know the same look&feel during the authoring. I think it's something related to the authoring habit. Somebody feel it's easier to use 2 columns table for authoring, and somebody prefer multiple columns. So far, grid type blocks like `Image-Collage` and `Cards` follow the 2 columns table, I think it's great to keep the consistency. 
Also, these variations are just introduced by healthy-thinking-report page, I don't feel it can bring much benefit to involve autoblocking. What do you think?

File: blocks/image-collage/image-collage.css [hunk 1]
```diff
@@ -14,6 +14,27 @@ main .image-collage>div {
   page-break-inside: avoid;
 }
 
+main .image-collage.icon-caption>div {
```
[the remark this answers is not in the document]
Comment (JiangLong2019) [reply blocks/image-collage/image-collage.css:17]: Okay, seems my IDE code formatter doesn't work, I'll check.

**Comment by dnbute** [comment]
> > Do you think we could replace the authoring experience for the first collage with something like this? We would also leverage the caption autoblocking this way and it might be easier for the authors to visualise what they're going for
> 
> @dnbute It's a good idea to let the author know the same look&feel during the authoring. I think it's something related to the authoring habit. Somebody feel it's easier to use 2 columns table for authoring, and somebody prefer multiple columns. So far, grid type blocks like `Image-Collage` and `Cards` follow the 2 columns table, I think it's great to keep the consistency. Also, these variations are just introduced by healthy-thinking-report page, I don't feel it can bring much benefit to involve autoblocking. What do you think?

Those are some good points as well, I think that the current image-collage is like that because that's what was needed at that point (I think it was created for the engineering website), I do agree that we should keep the experience consistent however. imho I think it's just a matter of opinion at this point, but yeah, since we have a pretty big number of blocks, consistency wins, let's keep it like this

File: blocks/image-collage/image-collage.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (JiangLong2019) [reply blocks/image-collage/image-collage.css]: @dnbute Addressed it. Please check.

File: blocks/image-collage/image-collage.css [hunk 2]
```diff
@@ -31,7 +52,56 @@ main .image-collage[class*='boxy-col-'].image-with-title > div {
...
+
+main .image-collage.icon-title-list > div > div li {
+  margin-bottom: 9px;
+  text-indent: -9px;
+  margin-left: 1rem;
+  line-height: 1.1rem;
+}
+
+main .image-collage.icon-title-list > div > div li::before {
+  content: "\2022";
```
Comment (dnbute) [inline blocks/image-collage/image-collage.css:96]: Can't we copy-paste the actual character?

**Review by dnbute — COMMENTED** [review]
Screenshot 2023-11-14 at 10 52 27

The last icon repeats a bunch of times, I might be wrong but this doesn't seem intended

File: blocks/image-collage/image-collage.css (hunk 2 above)
Comment (JiangLong2019) [reply blocks/image-collage/image-collage.css]: Done, you can check. 
(Personally, I prefer the Unicode to the bullet mark ^ ^)

**Comment by JiangLong2019** [comment]
@dnbute This was cached during my testing, I just clear the cache.
>The last icon repeats a bunch of times, I might be wrong but this doesn't seem intended

File: blocks/image-collage/image-collage.css (hunk 2 above)
Comment (dnbute) [reply blocks/image-collage/image-collage.css]: My thinking was that I had to google what the unicode meant, so why not just have it there anyway, but thanks for changing it 👍

**Review by dnbute — APPROVED** [review]
LGTM

**Pull request merged**